src/objects/Opportunity.object

```diff
@@ -855,6 +855,19 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Elevate_Recurring_ID_c__c</fullName>
+        <description>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</description>
+        <externalId>true</externalId>
+        <inlineHelpText>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</inlineHelpText>
+        <label>Elevate_Recurring_ID__c</label>
+        <length>255</length>
+        <required>false</required>
+        <trackFeedHistory>false</trackFeedHistory>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
```

Comment:
Should this be marked Unique?

Reply:
It will not be unique at the Opportunity level because a Recurring Donation can have more than one payment (Opp)

Reply:
No since this is actually the unique external ID of the parent record